.github/workflows/pr-e2e.yml

```diff
@@ -157,14 +168,7 @@ jobs:
           ENABLE_OPENTELEMETRY : true
           E2E_IMAGE_TAG: ${{ needs.triage.outputs.image_tag }}
           TEST_CLUSTER_NAME: keda-e2e-cluster-pr
-          COMMENT_BODY: ${{ github.event.comment.body }}
         run: |
-          MESSAGE="$COMMENT_BODY"
-          REGEX='/run-e2e (.+)'
-          if [[ "$MESSAGE" =~ $REGEX ]]
-          then
-            export E2E_TEST_REGEX="$(echo ${BASH_REMATCH[1]} | head -1)"
-          fi
```

Comment:
I think that we still need this code for filtering the regex during the e2e test execution